deploy/HelloWorld.deploy.ts

```diff
@@ -37,7 +37,11 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 export default func;
 
 func.skip = async (hre: HardhatRuntimeEnvironment) => {
-  const shouldSkip = hre.network.name === "ropsten";
+  const shouldSkip =
+    hre.network.name === "ropsten" ||
```

Comment:
also add this to the top so we sleep on these networks too before going ahead with the deployment